### PR TITLE
Added documentation and a few style fixes

### PR DESCRIPTION
**Pull request by aravindavk**
- Used `glustercsctl` for installation notes.
- Added documentation for Installation, File volumes, Block volumes and Thin arbiter volumes
- Used a drawing instead of screenshot for monitoring box in home page
- Preview is available in http://aravindavk.in/gcs

Signed-off-by: Aravinda VK <avishwan@redhat.com>

### CODE REVIEW

File: doc/block-volumes.md [hunk 1]
```diff
@@ -4,4 +4,115 @@ layout: doc
...
+Create a gluster virtual block storage class
+
+```yaml
+# File: cat storage-class.yaml
+---
+apiVersion: storage.k8s.io/v1
+kind: StorageClass
+metadata:
+  name: glustervirtblock-csi
+  annotations:
```
Comment (Madhu-1) [inline doc/block-volumes.md]: we can remove this annotation

File: doc/block-volumes.md [hunk 2]
```diff
@@ -4,4 +4,115 @@ layout: doc
...
+
+```
+[root@localhost]# kubectl get storageclass
+NAME                             PROVISIONER                    AGE
+glusterfs-csi (default)          org.gluster.glusterfs          28h
+glustervirtblock-csi             org.gluster.glustervirtblock   6s
+local-storage                    kubernetes.io/no-provisioner   29h
+```
+
+Persistent Volumes can be created using the storage class created
```
Comment (Madhu-1) [inline doc/block-volumes.md]: Persistent Volumes/ Persistent Volumes Claims

File: doc/block-volumes.md [hunk 3]
```diff
@@ -4,4 +4,115 @@ layout: doc
...
+```
+
+Verify gluster virtual block storage class
+
+```
+[root@localhost]# kubectl get storageclass
+NAME                             PROVISIONER                    AGE
+glusterfs-csi (default)          org.gluster.glusterfs          28h
+glustervirtblock-csi             org.gluster.glustervirtblock   6s
+local-storage                    kubernetes.io/no-provisioner   29h
```
Comment (Madhu-1) [inline doc/block-volumes.md]: remove other storage class than virt-block

File: doc/block-volumes.md [hunk 4]
```diff
@@ -4,4 +4,115 @@ layout: doc
...
+      storage: 1Gi
+```
+
+Run `kubectl create` command to submit the claim,
+
+```
+kubectl create -f pvc.yaml
+```
+
+Validate the File Volume or RWX claim creation by running `kubectl get
```
Comment (Madhu-1) [inline doc/block-volumes.md]: RWO or RWX

**Comment by aravindavk** [comment]
@Madhu-1  Addressed the review comments. Please review

File: doc/block-volumes.md (hunk 4 above)
Comment (Madhu-1) [reply doc/block-volumes.md]: @aravindavk ^^

File: doc/block-volumes.md [hunk 5]
```diff
@@ -4,4 +4,111 @@ layout: doc
 order: 4
 ---
 
-Block Volumes
+Create a gluster virtual block storage class
+
+```yaml
+# File: cat storage-class.yaml
```
Comment (Madhu-1) [inline doc/block-volumes.md]: sorry missed this one gcs deployment should create this SC

File: doc/installation.md [hunk 6]
```diff
@@ -4,4 +4,65 @@ layout: doc
...
+               nodes are supported.
+nodes        - Details of nodes where Gluster server pods needs to be
+               deployed.
+address      - Address of node as listed in kubectl get nodes(Use
+               `kubectl get nodes` to get the address)
+devices      - Raw devices which are required to auto provision
+               Gluster bricks during Volume create
+```
+
+Run the following command to deploy the GlusterCS cluster on a already
```
Comment (kotreshhr) [inline doc/installation.md]: nit: s/a already/an already

File: doc/quick-start-guide.md [hunk 7]
```diff
@@ -4,18 +4,119 @@ layout: doc
...
+               nodes are supported.
+nodes        - Details of nodes where Gluster server pods needs to be
+               deployed.
+address      - Address of node as listed in kubectl get nodes(Use
+               `kubectl get nodes` to get the address)
+devices      - Raw devices which are required to auto provision
+               Gluster bricks during Volume create
+```
+
+Run the following command to deploy the GlusterCS cluster on a already
```
Comment (kotreshhr) [inline doc/quick-start-guide.md]: same as above

File: pages/005-ease-of-use.md [hunk 8]
```diff
@@ -3,3 +3,22 @@ type: feature
...
+
+  - address: kube3
+    devices: ["/dev/vdc"]
+```
+
+```
+glustercsctl deploy ./mycluster.yml
+```
+
+Easy to install on a already running kubernetes cluster.
```
Comment (kotreshhr) [inline pages/005-ease-of-use.md]: same as above

File: doc/installation.md [hunk 9]
```diff
@@ -4,4 +4,65 @@ layout: doc
 order: 2
 ---
 
-Installation
+Install `glustercsctl` tool in one of the Kubernetes Master nodes
```
Comment (Madhu-1) [inline doc/installation.md]: any plan on moving `glustercsctl` under gluster org?

File: doc/quick-start-guide.md [hunk 10]
```diff
@@ -4,18 +4,119 @@ layout: doc
...
+`kubectl get pods -n <namespace>` command on the master node. For
+example,
+
+```
+kubectl get pods -n gcs
+```
+
+All the pods should be in Running state.
+
+To create a File Volume and a pod to use the volume,
```
Comment (Madhu-1) [inline doc/quick-start-guide.md]: I think this can be removed

**Pull request merged**